packages/core/src/dispatcher.ts

```diff
@@ -498,7 +497,7 @@ export class Dispatcher {
    * @param tip - Commit CID
    */
   publishTip(streamId: StreamID, tip: CID, model?: StreamID): Subscription {
-    if (process.env.CERAMIC_DISABLE_PUBSUB_UPDATES == 'true' || EnvironmentUtils.useRustCeramic()) {
+    if (process.env.CERAMIC_DISABLE_PUBSUB_UPDATES == 'true' || !this.enableSync) {
       return empty().subscribe()
```

Comment:
what is the difference between CERAMIC_DISABLE_PEER_DATA_SYNC and CERAMIC_DISABLE_PUBSUB_UPDATES ?

I take it this means we could just not publish, or we could not publish and not read or sync with other nodes?  but neither applies under rust ceramic?

If we were going to keep this i think it woudl be confusing but if we are going to leave js-ceramic entirely soon i guess it is fine.

Reply:
`CERAMIC_DISABLE_PUBSUB_UPDATES` is much more limited in scope.  It just disables publishing of update messages.  We'll still listen to pubsub and still publish query and response messages.

Honestly it's not really very useful, we put it in once a while ago to try to troubleshoot an issue gitcoin was having, I don't think anyone uses it anymore and we could frankly probably get rid of it.